internal/db/sqltest/tests/session/session_state_valid_transition_test.sql

```diff
@@ -0,0 +1,25 @@
+begin;
+    select plan(6);
+
+    -- Ensure session state table is populated
+    select is(count(*), 1::bigint) from session_state where session_id = 's1_____clare' and state='pending';
+    select is(count(*), 1::bigint) from session_state where session_id = 's1_____carly' and state='pending';
+
+    -- Valid state transition from pending to active
```

Comment:
similarly to the comment for the invalid tests, now that this pattern is set, can you add tests for the other valid transitions?